web/source/osk/oskManager.ts

```diff
@@ -1280,7 +1289,9 @@ namespace com.keyman.osk {
       if(device.formFactor == 'desktop') {
         Ls.position='absolute'; Ls.display='block'; //Ls.visibility='visible';
         Ls.left='0px';
-        this.loadCookie();
+
+        this.setSize();
+
```

Comment:
First commit only:  the 'kill-switch' to disable constant use of the OSK cookie.  The cookie would still be used to initialize the OSK, but any other control focuses would then rely on state fields.

As web cookies are synchronous, there shouldn't be any significant harm from continuing to rely on the cookies.